kimchi/src/circuits/polynomials/keccak/circuitgates.rs

```diff
@@ -99,25 +149,70 @@ where
 
         // LOAD STATES FROM WITNESS LAYOUT
         // THETA
-        let state_a = grid!(100, env.witness_curr_chunk(0, 100));
-        let shifts_c = grid!(80, env.witness_curr_chunk(100, 180));
-        let dense_c = grid!(20, env.witness_curr_chunk(180, 200));
-        let quotient_c = grid!(5, env.witness_curr_chunk(200, 205));
-        let remainder_c = grid!(20, env.witness_curr_chunk(205, 225));
-        let dense_rot_c = grid!(20, env.witness_curr_chunk(225, 245));
-        let expand_rot_c = grid!(20, env.witness_curr_chunk(245, 265));
+        let state_a = grid!(
+            100,
```

Comment:
I suppose here it is always the length.

Reply:
I don't know if you want to replace it.

Reply:
Yeah, it is always the length. But because of how the macro is defined, it does not recognize the pattern matching if I use a constant instead of the hard-coded number 😭